riff-raff/build.sbt

```diff
@@ -6,13 +6,15 @@ resolvers ++= Seq(
     "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
 )
 
+ivyScala := ivyScala.value map { _.copy(overrideScalaVersion = true) }
```

Comment:
Good question. I've added it because a warning told me to :grimacing:.

I think the reason for the warning is that different libraries try to pull in different versions of scala and this overrides that. There might be a better approach, but a search of guardian repos showed this was common practice.